### PR TITLE
Mapping to QUIC

### PR DESCRIPTION
**Pull request by aboba**
Fixes https://github.com/w3c/webtransport/issues/137

***
<a href="https://pr-preview.s3.amazonaws.com/aboba/webtransport/pull/176.html" title="Last updated on Feb 3, 2021, 1:39 AM UTC (ea1a397)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webtransport/176/b8d1a94...aboba:ea1a397.html" title="Last updated on Feb 3, 2021, 1:39 AM UTC (ea1a397)">Diff</a>

### CODE REVIEW

**Comment by aboba** [comment]
@vasilvv @ricea @jan-ivar Can you take a look?

**Comment by aboba** [comment]
Related to Issue https://github.com/w3c/webtransport/issues/94

File: index.bs [hunk 1]
```diff
@@ -838,6 +838,64 @@ interface ReceiveStream {
...
+      <tr>
+        <td>{{IncomingStream/readable}}.cancel()</td>
+        <td>send STOP_SENDING</td>
+      </tr>
+      <tr>
+        <td>{{IncomingStream/readable}}.getReader().read()</td>
+        <td>receive STREAM or STREAM_FINAL</td>
+      </tr>
+      <tr>
+        <td>{{IncomingStream/readable}}.getReader().cancel</td>
```
Comment (martinthomson) [inline index.bs:893]:
```suggestion
        <td>{{IncomingStream/readable}}.getReader().cancel()</td>
```

**Review by martinthomson — COMMENTED** [review]
I think that you need to make it clearer that these actions are not directly connected.  When you call one of these functions, you are interacting with a QUIC stack which will enqueue actions to send frames or pull data from a buffer of received frames.

File: index.bs (hunk 1 above)
Comment (aboba) [reply index.bs]: Fixed. However, now that the IETF WEBTRANS WG is going to focus on Http3Transport, the reference to [[QUIC-TRANSPORT]] is no longer appropriate.

**Comment by jan-ivar** [comment]
> I think that you need to make it clearer that these actions are not directly connected. When you call one of these functions, you are interacting with a QUIC stack which will enqueue actions to send frames or pull data from a buffer of received frames.

Similarly, on the upper end, we need to add algorithms interfacing with the streams spec https://github.com/w3c/webtransport/issues/137.

I worry a bit this table is brushing over details that need to be connected together correctly by implementers.

**Comment by jan-ivar** [comment]
That said, I'm not opposed to having both,  provided we mark this a non-normative.

**Comment by jan-ivar** [comment]
(I meant to link to https://github.com/w3c/webtransport/issues/185 earlier. Fixed)

**Comment by aboba** [comment]
Closing; updated in PR https://github.com/w3c/webtransport/pull/195

**Pull request closed without merging**